crates/typed-store/src/rocks/mod.rs

```diff
@@ -427,14 +427,22 @@ where
     }
 }
 
+/// Creates a default Rocksdb option.
+pub fn default_rocksdb_options() -> rocksdb::Options {
+    let mut opt = rocksdb::Options::default();
+    // Limit the total memtable memory usage per db.
+    opt.set_db_write_buffer_size(512 * 1024 * 1024);
```

Comment:
Please use a constant at the top of the file, e.g. `DB_WRITE_BUFFER_SIZE`.

Reply:
Good idea.